cypress/objects/creatTask.js

```diff
@@ -0,0 +1,23 @@
+class CreatTask {
+    getTaskNameField() {
+        return cy.get('[id="taskDetailsForm.name"]')
+    }
+
+    getDescriptionField() {
+        return cy.get('[id="taskDetailsForm.description"]')
+    }
+
+    getBillToClientSelect() {
```

Comment:
It's not clear what Select means. Would be better to include field type into the method name, i.e. "dropdown"